api.planx.uk/modules/webhooks/docs.yaml

```diff
@@ -113,12 +113,34 @@ components:
                           type: string
       required:
         - body
+      FlowStatusChange:
+        type: object
+        properties:
+          body:
+            type: object
```

Comment:
This body wrapper isn't required - changes to other events in the Swagger docs already on staging.